images/public-log-vendor-matcher/pg-init.d/lib-01-migrate-schemas.sql

```diff
@@ -0,0 +1,37 @@
+begin;
+
+create table if not exists cust_ip (
+  c_ip bigint not null
+);
```

Comment:
So here's my confusion: is this happening in BQ? Or in postgres?  I am very anti do-it-in-postgres when it comes to handling PII, because I don't want a stray container lying around with PII in it